test/foundry/dapp/Application.t.sol

```diff
@@ -46,18 +47,21 @@ contract ApplicationTest is ERC165Test {
         HelloWorld,
         MyOutput,
         ETHTransfer,
+        ETHMint,
```

Comment:
we could consider renaming these 2 outputs as: `ETHTransferEOA` `ETHTransferContract`

Reply:
Ok, I'll rename them.

Reply:
How about `ETHTransferNoPayload` and `ETHTransferWithPayload`?
I think `ETHTransferEOA` might be misleading, because the voucher `(recipient,value,"")` can be used for both EOAs and contract recipients. Likewise, a voucher `(recipient,value,payload)` (where `payload` is not empty) would also succeed if `recipient` were an EOA.

Reply:
How about `ETHTransfer` and `PayableFunction`?

Reply:
Because `ETHTransfer := (recipient,value,"")` can be used both for EOAs and contracts, so saying it's `ETHTransferEOA` might be misleading.

Reply:
In our setting `_recipient = LibBytes.hashToAddress("recipient");` is basically an EOA right?
But I do like `ETHTransfer` and `PayableFunc` much more, so let's do this ones :)

Reply:
Yes, because `_recipient` has no code by default, but we could inject code into it to test the scenario where the recipient is a contract.